idsvr/templates/deployment-admin.yaml

```diff
@@ -234,6 +234,10 @@ spec:
       {{- if .Values.curity.config.backup }}
       serviceAccountName: {{ include "curity.fullname" . }}-service-account
       {{- end }}
+      {{- with .Values.curity.admin.initContainers }}
+      initContainers:
+            {{- toYaml . | nindent 8 }}
+            {{- end }}
```

Comment:
Nit: usually `end` is indented to the same level as the opening block `with`.

Reply:
I prefer that too, but that's not the indent style used in this file. Though I'd be happy to adjust the indent in the rest of this file if that's something the maintainers are OK with.

Reply:
There are a couple of occurances where that formattign isn't done, fixes are welcomed!

Reply:
Fixed in a separate commit